<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaveModeWidget</class>
 <widget class="QWidget" name="LeaveModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QCheckBox" name="leaveModelCheck">
     <property name="text">
      <string>离开模式主开关</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="aiReplyPrivateCheck">
     <property name="text">
      <string>私聊AI回复（已失效）</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>回复最短间隔</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="aiReplyIntervalSpin">
       <property name="suffix">
        <string>秒</string>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="InteractiveButtonBase" name="aiReplyAllowButton">
     <property name="text">
      <string>开启自动回复的用户</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>前缀</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="aiReplyPrefixEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit
{
	background: transparent;
	border: none;
}</string>
        </property>
        <property name="placeholderText">
         <string>自动回复的前缀</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>后缀</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="aiReplySuffixEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit
{
    background: transparent;
    border: none;
}</string>
        </property>
        <property name="placeholderText">
         <string>自动回复的小尾巴</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>默认（没找到回复的情况下）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="aiReplyDefaultEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit
{
    background: transparent;
    border: none;
}</string>
        </property>
        <property name="placeholderText">
         <string>没有AI回复的情况下回复的固定内容，不带前后缀</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>此处回复暂用【神奇弹幕】的AI模型实现
可能会出现相关自称，不影响使用</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header>interactivebuttonbase.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
